src/core/thread/child_mask.hpp

```diff
@@ -36,11 +36,7 @@
 
 #include "openthread-core-config.h"
 
-#include <openthread/error.h>
-
-#include "common/code_utils.hpp"
-#include "common/debug.hpp"
-#include "common/encoding.hpp"
+#include "utils/bit_vector.hpp"
```

Comment:
I suggest we move `bit_vector.hpp` under `common` folder. The `common` is for all common helper modules used in OT core.

The  `utils` folder is mainly for extra features/functionalities that may not be directly part of Thread spec (and are not directly used by other OT core modules) but are useful additional/extra features we want to include in OT, for example, jam-detector, channel-monitor, heap managment, flash managment, etc.

Reply:
I moved it from `common/` to `utils/`. Now I feel sorry for that.